envs/environment.yaml

```diff
@@ -48,6 +48,7 @@ dependencies:
 - tqdm
 - pytz
 - jpype1
+- python-dotenv
```

Comment:
Add to envs/gb-model/workflow.yaml. We don't want to edit this file, to avoid merge issues if syncing with upstream.